CONTRIBUTING.md

```diff
@@ -1,14 +1,109 @@
-## Contributing to expressjs.com
+# Contributing to Expressjs.com
 
-This repository is only for issues related to the website [http://expressjs.com](http://expressjs.com). For issues related to Express, the framework, go to [https://github.com/expressjs/express](https://github.com/expressjs/express).
+### The Official Documentation of the Express JS Framework
 
-Feel free to make changes to the template files or the document files. The supporting docs are located in their respective directories, and the API docs are located under the `_includes` directory.
+>[!NOTE]
+> This is not the repo for Express JS. To contribute to the _[Express JS framework](https://github.com/expressjs/express)_ itself, [check this out](#were-the-express-js-frameworks-documenntation-team---not-the-express-js-framework).
 
-Content on this site is licensed under a Creative Commons Attribution-ShareAlike 3.0 United States License.  See https://creativecommons.org/licenses/by-sa/3.0/us/ for a layman's summary; 
-See [LICENSE.md](LICENSE.md) for the full license.
+If you want to contribute to [https://github.com/expressjs/expressjs.com](https://github.com/expressjs/expressjs.com), you're in the right place. 
 
+
+Whether you're a first time first time contributor or you're just looking for a refresher, visit our [Getting Started](#contributors-guide-to-getting-started) guide below to get up and running.
+
+#### Want to help but need some ideas? These are some of our most typical issue types:
```

Comment:
The reader is already here because they want to help. Change heading to simply "Need some ideas?" and move "These are some of our most typical issue types:" to be intro sentence.

Reply:
There wasn't any where to put "These are some..."  as an intro sentence so I left it in, but removed the block before that.